post-processing/fuzz_html.py

```diff
@@ -109,11 +109,24 @@ def create_horisontal_calltree_image(image_name: str, profile: fuzz_data_loader.
     plt.savefig(image_name)
     l.info("- image saved")
 
-def create_table_head(table_head: str, items: List[str]) -> str:
-    html_str = f"<table id='{table_head}' class='cell-border compact stripe'><thead><tr>\n"
+# Add table id and text in the th field here to add a title tag for it in the frontend.
+def get_th_hover_text(table_id: str, th_text: str) -> str:
+    tables = {"remaining_optimal_interesting_functions": {"bb count": "bb count"}}
```

Comment:
perhaps make the `items` list into a list of pairs where the first element in the pair is column name and the second element in the pair is the description

Reply:
> the dictionary here maps table ids to columns to descriptions? i.e. to add descriptions to another table one would have to extend it here?

Yes.

> Would it be possible to add the feature of column descriptions the same place we specify column names (the items argument)?

Yes, it would. The reason I did not do that is that it is a generalization that would support all `th` fields in the entire report, whereas currently the issue is with a handful of fields. I would suggest starting with support for just a few, selected `th` fields and extend it once this solution needs extending.

Reply:
> perhaps make the `items` list into a list of pairs where the first element in the pair is column name and the second element in the pair is the description

Yes, that would be the solution.

Reply:
> . I would suggest starting with support for just a few, selected `th` fields and extend it once this solution needs extending.

sounds good